lisa/features/__init__.py

```diff
@@ -1,7 +1,13 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-from .disks import DiskEphemeral, DiskPremiumLRS, DiskStandardLRS, DiskType
+from .disks import (
+    DiskEphemeral,
+    DiskPremiumLRS,
+    DiskStandardLRS,
```

Comment:
Maybe we should rename DiskStandardLRS to DiskStandardHDDLRS to differentiate scenario better. @LiliDeng @squirrelsc Thoughts?

Reply:
yep, I will make the change.